br/pkg/lightning/backend/local/duplicate.go

```diff
@@ -481,6 +488,11 @@ func (m *DupeDetector) RecordDataConflictError(ctx context.Context, stream DupKV
 		if err != nil {
 			return errors.Trace(err)
 		}
+
+		if algorithm == config.DupeResAlgErr {
+			return NewErrFoundConflictRecords(ctx, key, val, m.tbl)
```

Comment:
need to check with PM if we still need to save a record in downstream table.

And seems we can use `h.String()` and `m.decoder.DecodeRawRowDataAsStr(h, val)`? See line 499~500

Reply:
In `RecordIndexConflictError` we need to use `NewErrFoundConflictRecords` to decode the value since there is no value for `Row`
https://github.com/pingcap/tidb/blob/99390e2a0e3912df57e8ea2ed05cb24efb114d0e/br/pkg/lightning/backend/local/duplicate.go#L553-L557
So here I also use `NewErrFoundConflictRecords` to make them consistent

Reply:
I suppose for the ease of code, I can change to use `h.String()` and `m.decoder.DecodeRawRowDataAsStr(h, val)`